.github/workflows/on-pull-request.yml

```diff
@@ -33,3 +33,11 @@ jobs:
 
     - name: Build artifacts 🏗️
       run: npm run build
+
+    - name: Generate coverage 🧪
+      run: npm run test:ci
```

Comment:
Coverage should already be created when running `npm run build` (`"prebuild": "npm run test && rimraf dist/** && rimraf build/**"`). Please check if you can avoid running the tests twice.

Reply:
even better, looking into this